src/services/signingService.tsx

```diff
@@ -22,6 +22,7 @@ export interface SignerServiceSep10Request {
   challengeXDR: string;
   outToken: OutputTokenType;
   clientPublicKey: string;
+  address: string;
   memo?: boolean;
```

Comment:
Let's rename this parameter to something else. Just calling it `memo` makes it confusing when processing it on the signer-service. Let's call it something like

```suggestion
  usesMemo?: boolean;
```